tools/CMakeLists.txt

```diff
@@ -449,6 +471,17 @@ function(picotool_postprocess_binary TARGET)
     set_target_properties(${TARGET} PROPERTIES
         PICOTOOL_PROCESSING_CONFIGURED true
     )
+
+    # Allow using the example keys for pico-examples
+    if ((NOT DEFINED PICO_ALLOW_EXAMPLE_KEYS) AND (DEFINED PICO_EXAMPLES_PATH))
```

Comment:
I'm not sure if `(DEFINED PICO_EXAMPLES_PATH)` is the check that we want to use, as some people might set `-DPICO_EXAMPLES_PATH` unconditionally, as it's e.g. needed when you want to build the pico-sdk documentation. Would it be "better" to just `set(PICO_ALLOW_EXAMPLE_KEYS 1)` in the pico-examples `CMakeLists.txt` ?

Reply:
Yes, that sounds sensible, if `PICO_EXAMPLES_PATH` is being set for other projects